python/tvm/topi/x86/sparse.py

```diff
@@ -28,15 +28,17 @@ def schedule_sparse_dense(outs):
 
     def _callback(op):
         simd_width = get_fp32_len()
-        if op.tag == "sparse_dense_csrmm" and op != outs[0].op:
-            (_, v_i) = s[op].op.axis
-            s[op].vectorize(v_i)
-            (y_o, y_i) = s[outs[0].op].split(s[outs[0].op].op.axis[1], 2 * simd_width)
-            s[op].compute_at(s[outs[0]], y_o)
-            s[outs[0].op].vectorize(y_i)
-        if op.tag == "sparse_dense_bsrmm":
+        if op.tag == "sparse_dense_csrmm_v2" or op.tag == "sparse_dense_csrmm_v1":
+            (y_o, y_i) = s[op].split(s[op].op.axis[1], 2)
+            fused = s[op].fuse(s[op].op.axis[0], y_o)
+            s[op].parallel(fused)
+            s[op].vectorize(y_i)
+        elif op.tag == "sparse_dense_bsrmm_v2" or op.tag == "sparse_dense_bsrmm_v1":
```

Comment:
What is the difference between v1 and v2? Can we consolidate tags? If not, maybe we should rename them to be more clear about what they are doing.

Reply:
Basically the difference is whether lhs is sparse or rhs is sparse.
May be we change name like "sparse_dense_bsrmm_v2" --> "sparse_dense_bsrmm_sparse_rhs"  ?

Reply:
Yeah, lets do "sparse_dense_bsrmm_v2" --> "sparse_dense_bsrmm_sparse_rhs".